submissions/semenjob/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Secular+One&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style/style.css" />
+  </head>
+  <body>
+    <div class="bg">
+      <div class="container">
+        <div class="container_title">Music</div>
+        <div class="wrapper">
+          <div class="wrapper__list">
+            <div class="wrapper__item">Pop</div>
+            <div class="wrapper__item">K-Pop</div>
+            <div class="wrapper__item">Rock</div>
+            <div class="wrapper__item">Alternative Rock</div>
```

Comment:
Show more your skills in creating and drawing DOM elements.

Reply:
ok i understand what you mean